src/components/CreateEncounter/PatientPrescription.js

```diff
@@ -390,7 +391,7 @@ export const PatientPrescription = ({
                 }
 
                 return (
-                  <>
+                  <div key={getRandomKey()}>
```

Comment:
we can still use `Fragment` here. No need to add an extra `div`

Reply:
this fixes console error that there is no key